packages/ui/src/machines/authenticator/defaultAuthServices.ts

```diff
@@ -0,0 +1,36 @@
+import { Auth } from 'aws-amplify';
+export function signUp(username, password, attributes) {
+  return Auth.signUp({
+    username,
+    password,
+    attributes,
+    autoSignIn: {
+      enabled: true,
+    },
```

Comment:
This is where we default `autoSignIn` flag to `true`. 

https://docs.amplify.aws/lib/auth/emailpassword/q/platform/js/#auto-sign-in-after-sign-up